recipes/pickleDB/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pickleDB" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec6973e65d7d112849e78ce522840aa908efb2523470bb8ce5c7942310192240
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pickledb
+
+about:
+  home: http://github.com/patx/pickledb
+  license: BSD 3-clause
+  license_family: BSD
+  # There is currently no license file, see https://github.com/patx/pickledb/issues/52
+  # license_file:
```

Comment:
We need to ship a license file. Given that it says it is BSD 3-clause, we should either wait for upstream or add the file here in the recipe.